projects/packages/forms/src/blocks/field-telephone/edit.js

```diff
@@ -1,25 +1,176 @@
+import {
+	InspectorControls,
+	useBlockProps,
+	useInnerBlocksProps,
+	BlockContextProvider,
+	BlockControls,
+} from '@wordpress/block-editor';
+import {
+	PanelBody,
+	ToggleControl,
+	ToolbarButton,
+	ToolbarGroup,
+	SelectControl,
+} from '@wordpress/components';
+import { useCallback, useEffect, useMemo, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-import JetpackField from '../shared/components/jetpack-field';
+import { globe } from '@wordpress/icons';
+import clsx from 'clsx';
+import JetpackFieldControls from '../shared/components/jetpack-field-controls';
+import useFieldSelected from '../shared/hooks/use-field-selected';
 import useFormWrapper from '../shared/hooks/use-form-wrapper';
+import useJetpackFieldStyles from '../shared/hooks/use-jetpack-field-styles';
+import { countries } from './country-list';
+
+const EMPTY_ARRAY = [];
+
+const isBoolean = value => {
+	return value === true || value === false;
+};
+
+export default function PhoneFieldEdit( props ) {
+	const { setAttributes, attributes, clientId, isSelected } = props;
+	const {
+		showCountrySelector,
+		width,
+		id,
+		required,
+		requiredText,
+		placeholder,
+		default: defaultCountry,
+	} = attributes;
+	const [ countryList, setCountryList ] = useState( EMPTY_ARRAY );
+
+	const { isInnerBlockSelected, hasPlaceholder } = useFieldSelected( clientId );
+	const { blockStyle } = useJetpackFieldStyles( attributes );
+	const blockProps = useBlockProps( {
+		className: clsx(
+			'jetpack-field',
+			'jetpack-field-phone',
+			'jetpack-field-telephone',
+			width ? ` jetpack-field__width-${ width }` : '',
```

Comment:
Without space, since clx takes care of it?
```suggestion
			width ? `jetpack-field__width-${ width }` : '',
```

Reply:
Or I guess this would work too?

```suggestion
			{ `jetpack-field__width-${ width }`: width },
```